aten/src/ATen/native/cuda/ScaledGroupMM.cu

```diff
@@ -521,9 +499,294 @@ void dispatch_fp8_grouped_gemm_on_bias_dtype(
   }
 }
 
+} // namespace sm90_detail
+
+#if defined(BUILD_SM100_KERNEL)
+
+namespace sm100_detail {
+
+struct Sm100ConfigSmall {
+  using MmaTileShape = cute::Shape<cute::_64, cute::_128, cute::_128>;
+  using ClusterShape = cute::Shape<cute::_1, cute::_1, cute::_1>;
+  using KernelSchedule =
+      cutlass::gemm::KernelPtrArrayTmaWarpSpecializedBlockwise1SmSm100;
+  using EpilogueSchedule = cutlass::epilogue::PtrArrayTmaWarpSpecialized1Sm;
+
+  using ScaleConfig = cutlass::detail::Sm100BlockwiseScaleConfig<
+      /*SFVecSizeM*/ 1,
+      /*SFVecSizeN*/ 128,
+      /*SFVecSizeK*/ 128,
```

Comment:
The matrix dimensions N and K must be multiples of 128. The scaling factor layouts should follow the shape (M, K / 128) for scale_A and (N / 128, K / 128) for scale_B.

Reply:
Soo this appears to be the DeepGemm / deepseekv3 scaling? I think the PR body needs lots more description on this